### DISCUSSION TITLE
GitHub pages shows me a blank page

### DISCUSSION THREAD

**Discussion by Nonqs** [community/community#88596] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I just created a GitHub page and it only shows me a white page, if it suddenly helps, these are the errors it shows me

![image](https://github.com/community/community/assets/149440942/661fcf67-73ff-4aad-9464-50990bfd542a)
![a1baad62-b45a-4b84-890d-e48359b89ee9](https://github.com/community/community/assets/149440942/d7ebf77e-2cdd-4703-84b7-fc43e5b01f64)

**Comment by Shahnawazk7944** [comment]
Ensure that you have entered the correct URL. Check for any typos or misspellings.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @Nonqs 

This is because you are using React, on the GitHub pages there are some things you need to pay attention to so that it locates the assets correctly.

There is a discussion about this at this link below, reviewing these things should fix your problem:

https://github.com/orgs/community/discussions/60881#discussioncomment-6452616

And I recommend you use Vercel to deploy React applications, it already has a preset configuration for React applications and some other frameworks, just create an account at the link below with your GitHub and the rest is very intuitive:

https://vercel.com/

**Reply by Nonqs** [reply]
Ok, thank you!
